awscli/clidocs.py

```diff
@@ -335,6 +338,9 @@ def doc_option_example(self, arg_name, help_command, **kwargs):
     def doc_options_end(self, help_command, **kwargs):
         pass
 
+    def doc_global_option(self, help_command, **kwargs):
```

Comment:
Another thing I noticed is that with this change we should remove this logic as well: https://github.com/aws/aws-cli/blob/f39908a51c2843e146c56b2c2625b63f15312ad0/awscli/clidocs.py#L377-L384 that adds the `See 'aws help' for descriptions of global parameters.` since we are now documenting them.